Godeps/Godeps.json

```diff
@@ -147,18 +147,38 @@
 			"Rev": "bbeaab5ee38568daa84b8c233a8067b524588355"
 		},
 		{
+			"ImportPath": "github.com/containernetworking/cni/libcni",
+			"Comment": "v0.3.0-61-g9d5e6e6",
```

Comment:
They are using v0.4.0: https://github.com/kubernetes-incubator/cri-o/blob/v0.3/lock.json#L82.

Let's make sure we are using the same version.